src/models/good-oracle.model.ts

```diff
@@ -15,45 +16,52 @@ export class GoodOracle {
   /**
    *
    */
-  @property({required: true, jsonSchema: {
-  type: 'string',
-}})
-  id: string;
+  @property({
+    id: true,
+    required: true,
+    jsonSchema: {
+      type: 'number',
+    },
+  })
+  id: number;
```

Comment:
```suggestion
    jsonSchema: {
      type: 'string',
    },
  })
  id?: string;
```
given the new direction, that means id should be a string now
plus in order to create without requiring the presence of an id, we should remove the required = true and add the ? operator to allow for absence of this field.